scripts/assets/test_file_exemptions.textproto

```diff
@@ -543,6 +543,7 @@ exempted_file_path: "domain/src/main/java/org/oppia/android/domain/classify/rule
 exempted_file_path: "domain/src/main/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputRuleModule.kt"
 exempted_file_path: "domain/src/main/java/org/oppia/android/domain/classify/rules/ratioinput/RatioInputModule.kt"
 exempted_file_path: "domain/src/main/java/org/oppia/android/domain/classify/rules/textinput/TextInputRuleModule.kt"
+exempted_file_path: "domain/src/main/java/org/oppia/android/domain/devoptions/ShowAllHintsAndSolutionHelper.kt"
```

Comment:
Worth adding tests for, even if they're trivial. (We may complicate the impl in the future, so having a place to force testing considerations is generally good practice).

Reply:
Added test for `ShowAllHintsAndSolutionController`. PTAL. Thanks.